src/software/utils/main_sfmColorHarmonize.cpp

```diff
@@ -57,10 +57,8 @@ int aliceVision_main( int argc, char **argv )
       "Path to folder(s) in which computed matches are stored.")
     ("referenceImage", po::value<int>(&imgRef)->required(),
       "Reference image id.")
-    ("selectionMethod", po::value<int>(&selectionMethod)->required(),
-      "- 0: FullFrame\n"
-      "- 1: Matched Points\n"
-      "- 2: VLD Segment");
+    ("selectionMethod", po::value<EHistogramSelectionMethod>(&selectionMethod)->required(), 
+      "Histogram selection method");
```

Comment:
Add `EHistogramSelectionMethod_description()` with the list of options, else we cannot know the list of possible values.

Reply:
Done